test/types/range/nonIntTypes/colorRange.good

```diff
@@ -0,0 +1 @@
+colorRange.chpl:3: error: ranges of enum type are not currently supported
```

Comment:
Should the tests that result in this message be futures?

Would an enum range that turns out to be empty be an interesting test?  (e.g. ``color.blue..color.yellow``)

Reply:
They could be, but it seems equivalent to me and this approach requires fewer files.

Once enum ranges are supported it would be, but we'll want/need dozens more tests by that point.